.gitignore

```diff
@@ -4,7 +4,11 @@
 
 # Environment Specific #
 #####################
-/Projects/SeniorProject/devcycle-server/dataCollection/wsgi.py
+/dataCollection/wsgi.py
+/dataCollection/settings.py
```

Comment:
Spoke offline about why these are gitignored.  Will be making these as `THING.template` and call out what values need to change